### PR TITLE
[Merged by Bors] - chore(Order/Hom/Bounded): add `LE` instances

### PR DESCRIPTION
**Pull request by urkud**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Order/Hom/Bounded.lean [hunk 1]
```diff
@@ -307,6 +307,9 @@ theorem cancel_left {g : TopHom β γ} {f₁ f₂ : TopHom α β} (hg : Injectiv
 
 end Top
 
+instance [LE β] [Top β] : LE (TopHom α β) where
```
Comment (YaelDillies) [inline Mathlib/Order/Hom/Bounded.lean:310]:
```suggestion
instance instLE [LE β] [Top β] : LE (TopHom α β) where
```

File: Mathlib/Order/Hom/Bounded.lean [hunk 2]
```diff
@@ -500,6 +503,9 @@ theorem cancel_left {g : BotHom β γ} {f₁ f₂ : BotHom α β} (hg : Injectiv
 
 end Bot
 
+instance [LE β] [Bot β] : LE (BotHom α β) where
```
Comment (YaelDillies) [inline Mathlib/Order/Hom/Bounded.lean:506]:
```suggestion
instance instLE [LE β] [Bot β] : LE (BotHom α β) where
```

**Review by YaelDillies — APPROVED** [review]
maintainer merge

**Comment by j-loreaux** [comment]
bors merge

**Pull request closed without merging**